service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceService.kt

```diff
@@ -68,6 +71,7 @@ class InvoiceService(
             tx.updateInvoiceDates(succeeded.map { it.id }, invoiceDate, dueDate)
         }
         tx.setDraftsSent(succeeded.map { it.id to it.number!! }, user.evakaUserId)
+        tx.saveCostCenterFields(succeeded.map { it.id })
         tx.updateToWaitingForSending(withoutSSNs.map { it.id })
```

Comment:
Cost center information should be saved also for invoices that are manually sent (`withoutSSNs` here).